pyuvdata/uvdata/tests/test_ms.py

```diff
@@ -37,7 +37,13 @@ def test_cotter_ms():
 
     # check that a select on read works
     uvobj2 = UVData()
-    with uvtest.check_warnings(UserWarning, "Warning: select on read keyword set"):
+    # TODO: This is here as a catch for now - the way that MS dataset records baseline
+    # vectors appear to be rotated in the UV plane, such that V is oriented along the
+    # great circle of RA in the J2000/ICRF/requested reference frame (rather than the
+    # apparent coordinate plane). We should decide whether this is how pyuvdata coords
+    # should be oriented, or if they should be rotated so that "north" is towards the
+    # apparent celestial pole
```

Comment:
I imagine we'll get a chance to talk about this at some point, but FWIW, I originally came to this conclusion after playing around with some test MS/UVFITS data (can't remember which file at the moment) where I could clearly see said rotation present in the data. But just today, I stumbled upon this gem in the AIPS memo 117 (the AIPS SU discussion on page 36 of the 2021 Rev):
```
Coordinates. The source right ascension and declination at the chosen equinox are given by the contents
of the RAEPO and DECEPO columns while the chosen equinox is given by the EPOCH column. The values of
the projected baselines are rotated so that the images are in coordinates of equinox.
```
Funny that its so deeply buried in the memo, but I think it at least confirms what I had empirically demonstrated.

Reply:
Actually digging into this a bit further, it seems as though MIRIAD actually supports _both_ the "rotated" and "non-rotated" conventions (what they call current vs standard epoch, at least from what I was able to determine looking at docstrings inside the function `uvedit`), they just never bother to record which one is in the data file (which is why I feel less bad about adding the `obspa` keyword).  What things I've learned digging into ancient code...